src/app/MessageDef/StructBuilder.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+/**
+ *    @file
+ *      This file defines List builder in CHIP interaction model
```

Comment:
s/List/Struct

I generally advocate for removal of every single `@file` doccomment in our source code because errors are so easy in them (copy and paste) and they never seem to give much info beyond the obvious.

Reply:
removed @file